ci-operator/jobs/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-release-4.5-presubmits.yaml

```diff
@@ -347,7 +347,7 @@ presubmits:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-upgrade,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
I'd expect there to be a similar change in the `master` job file - did these diverge somehow?

Reply:
the master job file has `always_run: false`, so I changed the 4.5 job too